op-node/rollup/chain_spec.go

```diff
@@ -79,9 +80,9 @@ func (s *ChainSpec) MaxChannelBankSize(t uint64) uint64 {
 // ChannelTimeout returns the channel timeout constant.
 func (s *ChainSpec) ChannelTimeout(t uint64) uint64 {
 	if s.config.IsGranite(t) {
-		return s.config.ChannelTimeoutGranite
+		return params.ChannelTimeoutGranite
```

Comment:
We should return the config value in the not granite case and upstream of this, if the channel timeout is not set in the config then we should use the `params.ChannelTimeoutBedrock` as the default. This will reduce the amount of possible breaking changes with other chains, especially because we have tests in the superchain registry that may break due to this change, its best to not break the frontier chains, it will just cause a headache if we do

Reply:
Just to be clear, we should always use the hard coded value for granite, but we can use the configured value for bedrock if one is specified to preserve backwards compatibility.  We definitely shouldn't use `ChannelTimeoutBedrock` if granite is active.

Reply:
Alright. I'll do this. I'm not strongly against breaking backwards compatibility. But it's important to be clear on pre-granite settings, which we can do so in the standard config spec rather than break backwards compat.

Reply:
Though there won't be a default pre-granite channel timeout to keep it consistent with the current behavior.
It'll use whatever is specified in the config.